docs/contributors.md

```diff
@@ -1,40 +1,113 @@
 
-## Commands to use
+## How can I contribute...
 
-- npm run build - Compiles the action into a single js file at dist/index.js (Please check in the changes made by this command)
-- npm run test - Runs all tests under __tests__
-- npm run format - Runs formatting required to pass the lint test (Please check in the changes made by this command)
-- npm run update-installers - Updates the install-dotnet scripts in externals (Please check in the changes made by this command)
+* [Contribute Documentation:green_book:](#contribute-documentation)
 
-## To check in or not to check in
+* [Contribute Code :computer:](#contribute-code)
 
-- Do check in source (src)
-- Do check in index file (dist)
-- Do check in updates to install-dotnet scripts (externals)
-- Do not check in build output (lib)
-- Do not check in runtime (node_modules)
+* [Provide Support on Issues:pencil:](#provide-support-on-issues)
 
-## Writing tests
+* [Review Pull Requests:mag:](#review-pull-requests)
 
-With any contribution please take time to consider how this can be tested to maintain high quality. Current tests can be found in the folder __tests__ for examples.
+## Contribute documentation
 
-## Creating new version
+Documentation is a super important, critical part of this project. Docs are how we keep track of what we're doing, how, and why. It's how we stay on the same page about our policies and how we tell others everything they need to be able to use this project or contribute to it. 
 
-Details on versioning can be found here: https://github.com/actions/toolkit/blob/main/docs/action-versioning.md
-Create a new release using the UI. Version format should be `vX.Y.Z`. Creating a new major version requires reaction from users and should be done only with breaking changes.
-Once the new release is created, the vX tag needs to be updated as well.
-```
-git tag -fa vX -m "Update vX tag"
-git push origin vX --force
-```
+Documentation contributions of any size are welcome! Feel free to contribute even if you're just rewording a sentence to be more clear, or fixing a spelling mistake!
 
-For example, if you're publishing v2:
-```
-git tag -fa v2 -m "Update v2 tag"
-git push origin v2 --force
-```
\ No newline at end of file
+**How to contribute:**
+
+Pull requests are the easiest way to contribute changes to git repos at GitHub. They are the preferred contribution method, as they offer a nice way of commenting and amending the proposed changes.
+
+- Please check that no one else has already created a pull request with these changes
+- Use a "feature branch" for your changes. That separates the changes in the pull request from your other changes and makes it easy to edit/amend commits in the pull request 
+- Make sure your changes are formatted correctly and consistently with the rest of the documentation
+- Re-read what you wrote, and run a spellchecker on it to make sure you didn't miss anything
```

Comment:
Do we need to mention it ?

Reply:
Why not leave it as is. We should encourage people to check their spelling :)